### ISSUE TITLE
Generated Podfile done borked: undefined method `codegen_pre_install' for #<Pod::Podfile:0x0000>

### ISSUE THREAD

**Issue by skotchpine** [orYoffe/create-react-native-web-app#86]
**Describe the bug**
CRNWA added a method that wasn't available to the podfile.

**To Reproduce**
1. `npx crnwa xxx`, see `Error: Failed to install CocoaPods dependencies ...`
<img width="759" alt="Screen Shot 2020-11-02 at 11 45 18 AM" src="https://user-images.githubusercontent.com/13043909/97900852-e4288b80-1d00-11eb-8e68-9fe4d743dc66.png">

2. `cd xxx/ios; pod install`, see `undefined method codegen_pre_install ...`
<img width="572" alt="Screen Shot 2020-11-02 at 11 45 37 AM" src="https://user-images.githubusercontent.com/13043909/97900881-eee32080-1d00-11eb-9421-d7fe315de357.png">

**Desktop (please complete the following information):**
 - OS: macOS Catalina 10.15.7 (19H2)
- nodejs: v14.11.0
- npx: 6.14.8
 - crnwa: 0.16.1

**generated package.json**
```json
{
  "name": "xxx",
  "version": "0.0.1",
  "private": true,
  "scripts": {
    "android": "react-native run-android",
    "ios": "react-native run-ios",
    "start": "react-native start",
    "lint": "eslint .",
    "web": "react-scripts start",
    "build": "react-scripts build",
    "test": "react-scripts test",
    "eject": "react-scripts eject"
  },
  "dependencies": {
    "react": "17.0.1",
    "react-native": "0.63.2",
    "react-dom": "^17.0.1",
    "react-native-web": "^0.14.7",
    "react-scripts": "4.0.0",
    "web-vitals": "^0.2.4"
  },
  "devDependencies": {
    "@babel/core": "^7.8.4",
    "@babel/runtime": "^7.8.4",
    "@react-native-community/eslint-config": "^1.1.0",
    "metro-react-native-babel-preset": "^0.63.0",
    "react-test-renderer": "17.0.1",
    "react-native-codegen": "0.0.4"
  },
  "browserslist": {
    "production": [
      ">0.2%",
      "not dead",
      "not op_mini all"
    ],
    "development": [
      "last 1 chrome version",
      "last 1 firefox version",
      "last 1 safari version"
    ]
  }
}
```

**Comment by skotchpine** [comment]
I'm really just hacking around, and removing the method call is working fine for me right now

**Comment by orYoffe** [comment]
Hi @skotchpine,

Thanks for sharing the bug!
Could you please share more about your solution and how you solved it?
I believe it was permission control like the other issue you posted #87

**Comment by skotchpine** [comment]
@orYoffe Unfortunately, it was less of a solution and more of a workaround...

![Screen Shot 2020-11-04 at 1 53 15 PM](https://user-images.githubusercontent.com/13043909/98161294-1a067500-1ea5-11eb-8d63-c553bfd10cd6.png)

I'm glad to keep following up, but you should know that I decided to develop on top of `react-native init` and a hand-made webpack config to avoid customize-cra and other config complications.

**Comment by orYoffe** [comment]
Fixed in v0.16.2
